dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -56,7 +56,7 @@ RUN opam switch create "${COMPILER}+32bit" && eval $(opam env) && \
     opam install $BASE_OPAM
 
 # EDGE switch
-ENV COMPILER_EDGE="4.08.0" \
+ENV COMPILER_EDGE="4.08.1" \
```

Comment:
You did not change the `CACHEKEY`. Is that an overlook?

Reply:
Yes @Zimmi48 !! Sorry for that. Rebuilding both images.